yt/frontends/amrvac/data_structures.py

```diff
@@ -269,6 +274,8 @@ def _set_code_unit_attributes(self):
         else:
             # other case: numberdensity is supplied. Fall back to one (default) if no overrides supplied
             numberdensity_override = self.units_override.get('numberdensity_unit', (1, 'cm**-3'))
+            if 'numberdensity_unit' in self.units_override: # print similar warning as yt when overriding numberdensity
```

Comment:
If you're intentionally copying warning template here, this may make it harder to trace where the warning was emmited in case there's a bug with it or the code that it comes with

Reply:
Ah wow you reviewed before my comment :)
yeah so yt prints info to the console confirming the unit override, except for numberdensity.
I added this manually so it's consistent, otherwise you get an info message for eg. both length and temperature and not for numberdensity, it's a bit confusing

Reply:
So the rest of yt won't ever warn about overriding THIS quantity because it doesn't override it. Fair enough. But my point is that one may grep for `Overriding` or `mylog.info("Overriding`, and end up looking up the code in yt's core instead of here, so you might actually prefer to _break_ this warning's pattern to make this situation even less likely.

Reply:
So the rest of yt won't ever warn about overriding THIS quantity because it doesn't override it. Fair enough. But my point is that one may grep for Overriding or mylog.info("Overriding, and end up looking up the code in yt's core instead of here, so you might actually prefer to break this warning's pattern to make this situation even less likely.

Reply:
That's true, but personally I would prefer consistency over a small chance that grep gives an additional result. I was a bit confused myself as I gave `numberdensity_unit` together with length and temperature as an override, but yt printed that only the latter two were overridden with no info whatsoever on numberdensity. Yt's info messages are of this form 
`mylog.info("Overriding %s_unit: %g %s." ...`
while I explicitly put `numberdensity_unit` in the info message. Numberdensity is specific only to our frontend, yt does not support overriding that by default (but it's needed for amrvac). So actually, if people grep for it they should come out in the amrvac frontend, which is what we want, no?

Reply:
Ok, sold